pepper-apis/scripts/admin/delete-participant-data-for-study.sql

```diff
@@ -0,0 +1,183 @@
+-- scripts that will:
+-- generate a bunch of sql statements and shell script calls to be run to erase data for a study from dsm and dss.
+-- it also sets queued events to happen in the infinite future, effectively disabling them
+
+-- set the study to be deleted
+set @study_guid = 'singular' collate 'utf8mb4_unicode_ci';
+
+-- set the gcp environment in which to run the user deletes for dss
+set @gcp_project = 'broad-ddp-test';
```

Comment:
Is there a way to parameterize the env?
Seem like a new PR is required for each env?
Or do you expect this code is throw away after its use on each env?

Reply:
I am expecting that this script can be used to wipe out any study.  Just set these parameters and go, no need for a new PR.